.github/workflows/build.yml

```diff
@@ -108,6 +164,8 @@ jobs:
         run: NDK_CCACHE=1 NDK_CCACHE_BIN=ccache ./gradlew app:assemble${{ matrix.build_type }} --info
 
   build_ios:
+    # iOS is 10x expensive to run on GitHub machines, so only run if we know something else fast/simple passed as well
+    needs: build
```

Comment:
how this looks in practice 

![image](https://github.com/user-attachments/assets/0a5681a4-e8c0-4423-8268-e734b0909279)

Reply:
We could potentially move IOS to only run on main as well (see below for macos comment)

Reply:
> We could potentially move IOS to only run on main as well (see below for macos comment)

Wouldn't that cause problems for PRs? How would people know if their changes would break iOS/Mac?

Reply:
I think changing from Debug to Release will help a lot of build times. Also just making sure we are not running it unless others are passing is huge... so ya, maybe worth just keeping it for now